plugins/cisco_umbrella_reporting/help.md

```diff
@@ -98,27 +122,47 @@ _This plugin does not contain any triggers._
 
 ### Custom Output Types
 
+#### identity
+
+|Name|Type|Required|Description|
+|----|----|--------|-----------|
+|Deleted|boolean|False|A true/false flag indicating whether or not the identity is deleted|
+|ID|integer|False|The identifier of an identity|
+|Label|string|False|The label of an identity|
+
 #### record
 
 |Name|Type|Required|Description|
 |----|----|--------|-----------|
-|Action Taken|string|False|Either Blocked or Allowed|
+|All Applications|[]string|False|An array of all applications for entry|
+|Allowed Applications|[]string|False|An array of allowed applications for entry|
+|Blocked Applications|[]string|False|An array of blocked applications for entry|
 |Categories|[]string|False|Which categories, if any, the destination for which this request was made falls into|
-|Datetime|string|False|Datetime at which the event occurred, in UTC|
-|Destination|string|False|Destination to which this request was made|
+|Datetime|date|False|UTC Datetime at which the event occurred, represented in ISO 8601 format|
+|Domain|string|False|Domain to which this request was made|
 |External IP|string|False|External IP address of the identity making the request|
+|Identities|[]identity|False|An array of identities for entry|
 |Internal IP|string|False|Internal IP address of the identity making the request|
-|Origin Label|string|False|Human readable name for the identity, matches the one seen in the dashboard|
-|Origin Type|string|False|Identity type (such as network, roaming computer, AD User, etc)|
-|Origin ID|integer|False|Numerical identifier for the identity making the request|
-|Tags|[]string|False|Which tags, if any, the destination for which this request was made falls into|
+|Query Type|string|False|The type of DNS request that was made|
+|Threats|[]string|False|An array of threats for entry|
+|Timestamp|integer|False|The unix UTC timestamp in milliseconds|
+|Verdict|string|False|The entry verdict|
+
+#### threat
+
+|Name|Type|Required|Description|
+|----|----|--------|-----------|
+|Label|string|False|The threat label|
+|Type|string|False|Type of threat|
+
 
 ## Troubleshooting
 
-_This plugin does not contain any troubleshooting information._
+* The Cisco Umbrella Legacy API Keys are no longer used. Please use the Cisco Umbrella API Keys. In the links section it's described how to create/refresh/delete it.
 
 # Version History
 
+* 2.0.0 - Updated plugin to use API v2
```

Comment:
Suggestion - Add an additional note to say that OAuth authentication is now used and organization id is no longer required to setup the connection.